zboxcore/sdk/deleteworker.go

```diff
@@ -128,7 +147,7 @@ func (req *DeleteRequest) ProcessDelete() error {
 		return fmt.Errorf("Delete failed: %s", err.Error())
 	}
 
-	req.consensus = 0
+	req.consensus.consensus = float32(removedNum)
```

Comment:
And this place

Reply:
`delete` requests are done, and `commit` requests are not started here. it should be unnecessary to do it with lock here. do you think so?